src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -880,6 +885,32 @@ const Authenticate = (() => {
         }
     };
 
+    // This was added as a fix for onfido's default hash links which refresh the page and terminates
+    // the current verification flow. Disabled for now. Needed for 6.7.1 upwards
+    const mutationObserver = new MutationObserver((mutations) => {
+        mutations.forEach(() => {
+            const hash_links = $('a[href="#"]');
+            if (hash_links.length > 0) {
```

Comment:
do we still need this change?

Reply:
after sdk ver 6.7.2, apparently we don't need this anymore. removed 🙏🏼